modules/services/monitoring/prometheus-node-exporter.nix

```diff
@@ -0,0 +1,81 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib)
+    concatStringsSep
+    escapeShellArgs
+    getExe
+    mkEnableOption
+    mkIf
+    mkOption
+    mkPackageOption
+    types
+  ;
+
+  cfg = config.services.prometheus-node-exporter;
+in {
+  options = {
+    services.prometheus-node-exporter = {
+      enable = mkEnableOption "Prometheus Node exporter";
+
+      package = mkPackageOption pkgs "prometheus-node-exporter" { };
```

Comment:
No `package` option in NixOS (strangely?). I don’t consider the divergence a big deal, though, since NixOS should probably have it.

Reply:
Kept the package option as is.